packages/core/src/index.ts

```diff
@@ -62,7 +68,7 @@ export class Abby<
   Tests extends Record<string, ABConfig>
 > {
   private log = (...args: any[]) =>
-    this.config.debug ? console.log(`core.Abby`, ...args) : () => {};
+    this.config.debug ? console.log(`core.Abby`, ...args) : () => { };
```

Comment:
do we use `console`?

Reply:
We dont have a proper logger integrated right now , so is this used to log info in debugging mode atm.